src/engine/SCons/Action.py

```diff
@@ -95,7 +95,7 @@
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #
 
-__revision__ = "__FILE__ __REVISION__ __DATE__ __DEVELOPER__"
+__revision__ = "src/engine/SCons/Action.py 0.97p6 2015/11/19 17:09:57 robb"
```

Comment:
Sure. It looked deliberate but I can change it back in a few seconds…

From: Arnulfo Arroyo [mailto:notifications@github.com]
Sent: Thursday, August 04, 2016 4:16 PM
To: datalogics/scons
Cc: Michael McKeough; Author
Subject: Re: [datalogics/scons] Update the powershell change to make it apply to C# as well as Java. (#4)


In src/engine/SCons/Action.py<https://github.com/datalogics/scons/pull/4#discussion_r73604255>:

> @@ -95,7 +95,7 @@

>  # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.

>  #

>

> -__revision__ = "__FILE__ __REVISION__ __DATE__ __DEVELOPER__"

> +__revision__ = "src/engine/SCons/Action.py 0.97p6 2015/11/19 17:09:57 robb"

Can we keep the macro here as well?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/datalogics/scons/pull/4/files/2bb06b9cb58d83d452d19c535bdd4ad4541e8bf9#r73604255>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ABQVhaFn0Id5j2qcgyT5RH5eliiwNPMBks5qclaTgaJpZM4JdEVh>.